### PR TITLE
Add getOperands function

### PR DESCRIPTION
**Pull request by kael89**
May be also nice to replace the operator symbols (`+`, `-` etc) with symbolic constants, I may attempt it in another PR if it's not too tricky

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,27 @@
 
 Utility to evaluate [BODMAS](https://en.wikipedia.org/wiki/Order_of_operations) arithmetic formulas. It is an implementation of the [shunting yard algorithm](https://en.wikipedia.org/wiki/Shunting-yard_algorithm).
 
-### Installation
+## Installation
 
+With `yarn`:
+
+```
+yarn add @beyondessential/arithmetic
```
Comment (kael89) [inline README.md:10]: Installation instructions are updated since we'll publish this to `npm`

File: src/symbols.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      return 3;
+    case '+':
+    case '-':
+      return 2;
+    default:
+      throw new Error('Invalid operator');
+  }
+}
+
+export function getOperands(formulaText) {
```
Comment (kael89) [inline src/symbols.js:28]: Two questions about the specifications of this function:

1. Should it validate the formula first? (eg checking for umatched parentheses etc).
2. Should it parse numbers as integers in the result? Eg `getOperands('a + 1') // ['a', 1], currently: ['a', '1']`

I'd say yes for 1, and probably for 2 too

Now that I think of it, perhaps we should expose a validator function in the package's API

File: index.d.ts [hunk 3]
```diff
@@ -1 +1,12 @@
-export function runArithmetic(formula: string, values?: Record<string, string | number>): number;
+/**
+ * Copyright (c) 2020 Beyond Essential Systems Pty Ltd
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export function getOperand(formulaText: string): string[];
```
Comment (edmofro) [inline index.d.ts:8]:
```suggestion
export function getOperands(formulaText: string): string[];
```

File: src/symbols.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  const codes = formulaText
+    // Replace the alternate multiplication symbol 'x' with a non-alphanumeric character
+    .replace(/(^|\W)x(\W|$)/, ' ')
+    .split(/[+-/*() ]/g)
+    .filter(c => c !== '');
+
+  return [...new Set(codes)];
```
Comment (edmofro) [inline src/symbols.js:35]:
```suggestion
  const operands = formulaText
    // Replace the alternate multiplication symbol 'x' with a non-alphanumeric character
    .replace(/(^|\W)x(\W|$)/, ' ')
    .split(/[+-/*() ]/g)
    .filter(o => o !== '');

  return [...new Set(operands)];
```

File: src/symbols.js (hunk 2 above)
Comment (edmofro) [reply src/symbols.js:28]: I could go either way on both of those points, so I'm not much help!

**Review by edmofro — CHANGES_REQUESTED** [review]
Just a couple of tweaks - looks good though

File: src/symbols.js (hunk 2 above)
Comment (kael89) [reply src/symbols.js:28]: They are not critical at this stage so I'll leave them for another version

**Review by edmofro — APPROVED** [review]

**Pull request merged**